### ISSUE TITLE
Taiga-events could really benefit from having some logs!

### ISSUE THREAD

**Issue by herbsmn** [taigaio/taiga-events#12]
Please allow some logs to be created for Taiga-events.

**Comment by juanfran** [comment]
You're completely right. I've created an enhacement with your request https://tree.taiga.io/project/taiga/issue/4397

We'll try to solve it as soon as possible, thanks

**Comment by herbsmn** [comment]
many thanks
